src/main/java/com/hariharanweb/remoteappiummanager/Server.java

```diff
@@ -43,6 +43,14 @@ public static void main(String[] args) throws IOException {
             });
         });
 
+        path("/startadblog", () -> {
```

Comment:
Can we change the path as : '/devices/adblog/start' & '/device/adblog/stop/:udid' ?

Reply:
Ya I will update the paths